packages/features/form-builder/schema.ts

```diff
@@ -308,12 +309,12 @@ export const fieldTypesSchemaMap: Partial<
         }
         const valueIdentified = response as unknown as Record<string, string>;
         if (subField.required) {
+          if (!isPartialSchema && !valueIdentified[subField.name])
```

Comment:
Fixed by @hariombalhara.

Invalid String error should be the fallback. We don't want such error if the subField is required and not filled.